modules/common/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/registry/JavascriptScriptEnrichment.scala

```diff
@@ -54,21 +55,27 @@ object JavascriptScriptEnrichment extends ParseableEnrichment {
       _ <- isParseable(c, schemaKey)
       encoded <- CirceUtils.extract[String](c, "parameters", "script").toEither
       script <- ConversionUtils.decodeBase64Url(encoded)
+      params <- CirceUtils.extract[Option[JsonObject]](c, "parameters", "config").toEither
       _ <- if (script.isEmpty) Left("Provided script for JS enrichment is empty") else Right(())
-    } yield JavascriptScriptConf(schemaKey, script)).toValidatedNel
+    } yield JavascriptScriptConf(schemaKey, script, params.getOrElse(JsonObject.empty))).toValidatedNel
 }
 
-final case class JavascriptScriptEnrichment(schemaKey: SchemaKey, rawFunction: String) extends Enrichment {
+final case class JavascriptScriptEnrichment(
+  schemaKey: SchemaKey,
+  rawFunction: String,
+  params: JsonObject = JsonObject.empty
+) extends Enrichment {
   private val enrichmentInfo =
     FailureDetails.EnrichmentInformation(schemaKey, "Javascript enrichment").some
 
   private val engine = new ScriptEngineManager(null)
     .getEngineByMimeType("text/javascript")
     .asInstanceOf[ScriptEngine with Invocable with Compilable]
 
-  private val stringified = rawFunction + """
+  private val stringified = rawFunction + s"""
+    var _params = ${params.asJson.noSpaces};
```

Comment:
Just for fun, I played with whether it's possible to hide the `_params` from the global namespace: #841